src/components/settings/data/hooks.js

```diff
@@ -132,13 +132,7 @@ export const useStylesForCustomBrandColors = (branding) => {
       .btn-brand:focus:before {
         border-color: ${brandColors.secondary.regular.hex()} !important;
       }
-      .btn-primary {
-        background-color: ${brandColors.primary.regular.hex()} !important;
-        border-color: ${brandColors.primary.regular.hex()} !important;
-        color: ${brandColors.primary.textColor.hex()} !important;
-      }
       .btn-primary:hover {
-        background-color: ${brandColors.primary.dark.hex()} !important;
```

Comment:
this was overriding the styles for the buttons in `SearchPagination` component. I removed this and it doesn't look like it affected other areas of the app. 

[question] should we be using hooks to override styles and is this the best approach?